### PR TITLE
feat: Radix UI payments

### PR DESCRIPTION
**Pull request by mkleszcz**
### Please check if the PR fulfills these requirements

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)

### What kind of change does this PR introduce?

Updated design of the payments and subscription page (WIP)

### Does this PR introduce a breaking change?

Nope.

### Other information:

image
image
image

### CODE REVIEW

**Review by kacperadler — APPROVED** [review]

File: packages/webapp-libs/webapp-finances/src/components/stripe/stripeCardForm/stripeCardForm.component.tsx [hunk 1]
```diff
@@ -24,10 +18,10 @@ type StripeFieldProps = {
 };
 
 const StripeField = ({ children, label, small }: StripeFieldProps) => (
```
Comment (bielsky0) [inline packages/webapp-libs/webapp-finances/src/components/stripe/stripeCardForm/stripeCardForm.component.tsx:20]: `small` prop isn't used

File: packages/webapp-libs/webapp-finances/src/routes/subscriptions/subscriptions.component.tsx [hunk 2]
```diff
@@ -1,43 +1,65 @@
+import { PageHeadline } from '@sb/webapp-core/components/pageHeadline';
+import { PageLayout } from '@sb/webapp-core/components/pageLayout';
+import { Separator } from '@sb/webapp-core/components/separator';
 import { FormattedMessage } from 'react-intl';
 
 import { useActiveSubscriptionDetails } from '../../components/activeSubscriptionContext';
 import { PaymentMethodContent } from './paymentMethod.content';
 import { SubscriptionsContent } from './subscriptions.content';
-import { Container, Header, Section, Subheader } from './subscriptions.styles';
```
Comment (bielsky0) [inline packages/webapp-libs/webapp-finances/src/routes/subscriptions/subscriptions.component.tsx:6]: I think we can remove this style file

**Review by bielsky0 — CHANGES_REQUESTED** [review]
CancelSubscription view still uses styled-components

File: packages/webapp-libs/webapp-finances/src/routes/subscriptions/subscriptions.component.tsx (hunk 2 above)
Comment (mkleszcz) [reply packages/webapp-libs/webapp-finances/src/routes/subscriptions/subscriptions.component.tsx:6]: This file contain styles that are used by SubscriptionsContent component which is not done yet

**Comment by mkleszcz** [comment]
> CancelSubscription view still uses styled-components

@bielsky0 This PR changes mainly payments and some parts of the subscriptions (mainly the shared parts between payments and subscriptions). I will adjust rest of the subscriptions screens later.

**Review by bielsky0 — APPROVED** [review]

**Pull request merged**